src/main/java/org/broadinstitute/dsm/route/LoggingFilter.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.dsm.route;
+
+import java.util.Map;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.exceptions.JWTDecodeException;
+import com.auth0.jwt.impl.JWTParser;
+import com.auth0.jwt.interfaces.Claim;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.util.Utility;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.log4j.MDC;
+import spark.Filter;
+import spark.Request;
+import spark.Response;
+
+/**
+ * Captures client IP and name of user from token, adding
+ * them to MDC for logging.
+ */
+public class LoggingFilter implements Filter {
```

Comment:
Was surprised  that we  aren't logging IP and JWT user id info--so I added it, since this is an important part of compliance.